### DISCUSSION TITLE
How to use VSCode debugger in a turbo monorepo?

### DISCUSSION THREAD

**Discussion by p0mvn** [vercel/turborepo#7135] [Help]
Hi everyone! I'm looking for a VS code configuration to be able to set up [this project](https://github.com/osmosis-labs/osmosis-frontend) with a debugger.

I've tried a number of configurations but I'm unable to hit breakpoints even if the debugger is started.

The closest one I got to working is the following launch.json:
```
        {
            "name": "Next.js: debug server-side",
            "type": "node-terminal",
            "request": "launch",
            "command": "yarn run dev",
            "skipFiles": [
                "<node_internals>/**"
            ],
            "env": {
                "NODE_OPTIONS": "--inspect"
              }
          },
```

Where the dev script runs `turbo dev`

In the logs, I see "address already in use"
![image](https://github.com/vercel/turbo/assets/34196718/cc241022-be28-4a7a-aa46-fdf744096d50)

I suspect that each package attempts to use the same debugger port.

Would anyone please be able to direct me to docs on how I could set up a debugger with VS code for my project.

Thank you!

**Comment by gethari** [comment]
Did you get this working ?
